src/test/java/com/faforever/client/fx/contextmenu/AbstractCustomMenuItemControllerTest.java

```diff
@@ -0,0 +1,29 @@
+package com.faforever.client.fx.contextmenu;
+
+import com.faforever.client.test.UITest;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class AbstractCustomMenuItemControllerTest extends UITest {
+
+  private AbstractCustomMenuItemController<Object> instance;
```

Comment:
If all the concrete classes are tested we don't really need this but that is okay